teku/src/test/java/tech/pegasys/teku/cli/subcommand/RemoteSpecLoaderTest.java

```diff
@@ -55,6 +64,29 @@ void shouldFailWhenRequiredItemsAreMissing() {
         .hasMessageContaining("GENESIS_FORK_VERSION");
   }
 
+  @Test
+  void shouldProvideValidSpecConfigWithIncompleteRemoteConfig() throws IOException {
+    final String jsonConfig =
+        Resources.toString(
+            Resources.getResource(RemoteSpecLoaderTest.class, "spec_config.json"),
+            StandardCharsets.UTF_8);
+    final ObjectMapper objectMapper = new ObjectMapper();
+    TypeReference<Map<String, String>> typeReference = new TypeReference<>() {};
+    Map<String, String> data = objectMapper.readValue(jsonConfig, typeReference);
+
+    final SpecConfig specConfig = SpecConfigLoader.loadRemoteConfig(data);
+
+    final SpecConfig expectedConfig =
+        TestSpecFactory.getDenebSpecConfig(
+            Eth2Network.MAINNET,
+            UInt64.valueOf(74240),
+            UInt64.valueOf(144896),
+            UInt64.valueOf(194048),
+            UInt64.valueOf("18446744073709551615"),
+            false);
+    assertThat(specConfig).isEqualTo(expectedConfig);
```

Comment:
rather than comparing the entire spec, all we need to know is that the load didnt cause an exception (it couldnt have to get this far) and that the fields that were missing are now set with the correct values. so we can check max_chunk_size etc explicitly from our mainnet spec we internally generate against the one we load remotely.